lib/qb_integration/vendor.rb

```diff
@@ -22,19 +22,26 @@ def index
 
     def create
       vendor = vendor_service.create
-      [200 , "Vendor with id #{vendor.id} created", as_flowlink_hash(vendor)]
+      updated_flowlink_vendor = payload[:vendor]
+      updated_flowlink_vendor[:qbo_id] = vendor.id
+      [200 , "Vendor with id #{vendor.id} created", updated_flowlink_vendor]
     end
 
     def update
-      vendor = vendor_service.update
-      [200 , "Vendor with id #{vendor.id} updated", as_flowlink_hash(vendor)]
+      vendor, action = vendor_service.update
+      puts vendor
```

Comment:
Are these puts for logging?

Reply:
aghh good catch, let me remove haha

Reply:
ideally we would have some kind of EnpointBase.log("whatever") or even request.log("whatever"), and the endpoint base gem would generate a JSON based logging with all info